bikeshed/boilerplate.py

```diff
@@ -450,7 +449,9 @@ def disambiguator(ref):
         refsFromText = defaultdict(list)
         for ref in possibleRefs:
             for i, existingRef in enumerate(refsFromText[ref.text]):
-                if disambiguator(existingRef) != disambiguator(ref):
+                if disambiguator(existingRef, types, specs) != disambiguator(
+                    ref, types, specs
```

Comment:
What's causing this sort of line-length protection to still happen? A different part of the formatting pipeline using a smaller limit than I specified in flake8?

Reply:
This was a warning/error from Black when I first pushed